### ISSUE TITLE
App misbehaving when creating the preferences pane

### ISSUE THREAD

**Issue by HalfVoxel** [martin31821/cpupower#193]
# Creating a new issue

The `buildPrefsWidget` just spawns a separate process with the actual preferences. This is bad because there will be a leftover non-responding application. If you focus this leftover application using alt-tab it shows a "This application is not responding" dialog which just looks bad.

Gnome shell: 42.2
![image](https://user-images.githubusercontent.com/1144597/175808874-24602bfd-d2b3-4128-9d8e-cfc5e0d88288.png)

**Comment by fin-ger** [comment]
Yep, this is not really elegant. But with a GTK4 rework, this issue will become irrelevant. Therefore, the current version will not be fixed.
